django/db/backends/postgresql/client.py

```diff
@@ -22,13 +22,17 @@ def settings_to_cmd_args_env(cls, settings_dict, parameters):
         sslcert = options.get('sslcert')
         sslkey = options.get('sslkey')
 
+        if not dbname and not service:
+            # Connect to the default 'postgres' db.
+            dbname = 'postgres'
         if user:
             args += ['-U', user]
         if host:
             args += ['-h', host]
         if port:
             args += ['-p', str(port)]
-        args += [dbname]
+        if dbname:
+            args += [dbname]
```

Comment:
Let's be explicit and use the `-d` flag:

```suggestion
            args += ['-d', dbname]
```

(Actually it'd be nice to use `--dbname` here as well as `--username`, `--host`, and `--port`, but that is for another PR... Using longer flag names is self-documenting and should be encouraged.)

Reply:
Added.

Reply:
I reverted this change, I don't want to mix bugfix with unrelated amendments to `dbshell` tests.